vcd_cli/role.py

```diff
@@ -28,7 +28,7 @@ def role(ctx):
 def list_roles(ctx):
     try:
         client = ctx.obj['client']
-        org_name = ctx.obj['profiles'].get('org_in_use')
```

Comment:
I will prefer to figure out if the logged in user is admin or not in some other (more direct) fashion. The current approach feels a bit convoluted.

Reply:
Yes, the current `is_admin` flag can be confusing. We can add a method in the `client` class